install_pim/customize_dataset/index.rst

```diff
@@ -10,6 +10,12 @@ Akeneo PIM ships with *icecat-demo-dev* enabled.
 Switching From Icecat to Minimal
 --------------------------------
 
+Edit the file ``app/config/pim_parameters.yml``
```

Comment:
Don't we advise to modify `parameters.yml` usually ?

Reply:
Yes right

Reply:
Aggreed. One should not have to touch this file.

Reply:
@fitn @phaseinducer @damien-carcel Let me clear things... I'm sure we need to change `parameters.yml` instead of `pim_parameters.yml`
But in community standard repo it is not configurable and `parameters.yml` is not too.

https://github.com/akeneo/pim-community-standard/blob/master/app/config/parameters.yml.dist

https://github.com/akeneo/pim-community-standard/blob/master/app/config/pim_parameters.yml

https://github.com/akeneo/pim-community-standard/blob/master/app/config/config.yml